icu4c/source/common/uloc_keytype.cpp

```diff
@@ -222,11 +222,6 @@ initFromResourceBundle(UErrorCode& sts) {
                 if (U_FAILURE(sts)) {
                     break;
                 }
-                // TODO: For now we skip loading entries mapping to metazone short ids,
```

Comment:
Revert the addition of code to exclude 4-character metazone short IDs added in the previous integration PR #1843, since those IDs are now moved to a different table